src/test/java/com/deloitte/elrr/services/controller/CommonControllerTest.java

```diff
@@ -37,9 +40,7 @@
  * @author mnelakurti
  *
  */
-@Getter
-@Setter
-@NoArgsConstructor
+@Getter @Setter @NoArgsConstructor
```

Comment:
looks like the autoformatter is doing this :/

Reply:
yeah please god no. this was what i was referencing earlier.

Reply:
changed back